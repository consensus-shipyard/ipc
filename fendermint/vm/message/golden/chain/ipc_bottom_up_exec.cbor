a163497063a16c426f74746f6d557045786563a2676d657373616765a4697375626e65745f6964821ba1656b9a6ab736e08156040a888c4e06372e45158764a4a11d1ce500f820527a666865696768741a6f9a42ea756e6578745f76616c696461746f725f7365745f69641bbc4d488f39d39de672626f74746f6d5f75705f6d65737361676573d82a583e0001bb1890e30a36e50c4b971c3b52ec39ea0c120cc36c6214347e5b4f3428218d54bbfb42a5cf602e01aff7cfd8ac07f856dccec3275ca245e676ed16ae6b6365727469666963617465a16a7369676e61747572657385a26976616c696461746f724a00d2a4e297d084abea14697369676e61747572654101a26976616c696461746f724a00d8d3eac7c4afefcb61697369676e61747572654102a26976616c696461746f724a00eefad6e88299a7ae46697369676e617475726546020101ffa8f6a26976616c696461746f7255020a01ce813bce190139aef784736d2800e38b3d9c697369676e6174757265420277a26976616c696461746f725501a1c9bf8d8cb7568dc5f50cff4d9a4facd855d0de697369676e617475726543020172