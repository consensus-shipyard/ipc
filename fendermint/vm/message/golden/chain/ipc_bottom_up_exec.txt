Ipc(BottomUpExec(CertifiedMessage { message: BottomUpCheckpoint { subnet_id: SubnetID { root: 11629819923713701600, children: [Address { payload: Delegated(DelegatedAddress { namespace: 10, length: 20, buffer: [136, 140, 78, 6, 55, 46, 69, 21, 135, 100, 164, 161, 29, 28, 229, 0, 248, 32, 82, 122, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }] }, height: 1872380650, next_validator_set_id: 13568581032324865510, bottom_up_messages: Cid(bag5rrehdbi3okdcls4odwuxmhhvayeqmynwgefbupznu6nbieggvjo73iks46yboagx7pt6yvqd7qvw4z3bsoxfcixthn3iwvy) }, certificate: MultiSig { signatures: [ValidatorSignature { validator: Address { payload: ID(1501013850784830034) }, signature: Signature { sig_type: Secp256k1, bytes: [] } }, ValidatorSignature { validator: Address { payload: ID(7032297684660300248) }, signature: Signature { sig_type: BLS, bytes: [] } }, ValidatorSignature { validator: Address { payload: ID(5070099664076127598) }, signature: Signature { sig_type: BLS, bytes: [1, 1, 255, 168, 246] } }, ValidatorSignature { validator: Address { payload: Actor([10, 1, 206, 129, 59, 206, 25, 1, 57, 174, 247, 132, 115, 109, 40, 0, 227, 139, 61, 156]) }, signature: Signature { sig_type: BLS, bytes: [119] } }, ValidatorSignature { validator: Address { payload: Secp256k1([161, 201, 191, 141, 140, 183, 86, 141, 197, 245, 12, 255, 77, 154, 79, 172, 216, 85, 208, 222]) }, signature: Signature { sig_type: BLS, bytes: [1, 114] } }] } }))